src/Compilers/CSharp/Portable/Symbols/Source/LambdaSymbol.cs

```diff
@@ -47,7 +47,7 @@ internal sealed class LambdaSymbol : SourceMethodSymbol
             _messageID = unboundLambda.Data.MessageID;
             _syntax = unboundLambda.Syntax;
             _refKind = refKind;
-            _returnType = returnType ?? ReturnTypeIsBeingInferred;
+            _returnType = returnType == null ? TypeSymbolWithAnnotations.CreateUnannotated(NonNullTypesFalseContext.Instance, ReturnTypeIsBeingInferred) : returnType;
```

Comment:
>NonNullTypesFalseContext.Instance [](start = 91, length = 33)

Can we pass a better context here too?

Reply:
Placeholder, for `TypeSymbol` only.

---
In reply to: [206365931](https://github.com/dotnet/roslyn/pull/28943#discussion_r206365931) [](ancestors = 206365931)